frb_example/pure_dart/rust/src/api.rs

```diff
@@ -584,3 +584,27 @@ pub fn create_event() {
         });
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct Log {
+    pub key: u32,
+    pub value: u32,
+}
+
+pub fn handle_stream_1(key: u32, max: u32, sink: StreamSink<Log>) -> Result<(), anyhow::Error> {
```

Comment:
Shall we give handle_stream_1, 2, and 3 a more meaningful name. For example, "handle_stream_sink_at_0, _sink_at_1, etc"